cogboard-webapp/src/components/FormField/helpers.js

```diff
@@ -0,0 +1,26 @@
+import { splitPropsGroupName } from '../../utils/components';
+
+export const camelToKebab = string =>
```

Comment:
I love this name 💯 I'm imagining a camel entering some kind of a machine and then from machine Kebabs are coming out.